vendor/bat-native-ads/src/bat/ads/internal/creatives/notification_ads/creative_notification_ads_database_table.cc

```diff
@@ -42,6 +44,19 @@ constexpr char kTableName[] = "creative_ad_notifications";
 
 constexpr int kDefaultBatchSize = 50;
 
+std::vector<float> ConvertStringToVector(std::string string) {
```

Comment:
This seems like it should be defined at a higher level so other functions can use it

Reply:
Could we not use `base::SplitString`?

Reply:
Could we not use base::SplitString, there maybe other more suitable base utils